Run Google App Engine on Fedora (python)

```diff
@@ -0,0 +1,27 @@
+I faced a lot of challenges while making Google App Engine run on my Fedora platform. So, to tackle the exact problems faced by me, I am writing this post:
+
+STEP 1:
```

Comment:
I am unable to bold it.I googled even, but no satisfying answer I got. Can you help me in this?